### PR TITLE
DAOS-12647 doc: Update LED instructions with custom timeout

### PR DESCRIPTION
**Pull request by tanabarr**
Correct instructions regarding specifying LED identify time period by
removing references to environment variable and instead recommending
the use of --timeout dmg option.

Required-githooks: true

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [x] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [x] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [x] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [x] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

File: docs/admin/administration.md [hunk 1]
```diff
@@ -516,12 +516,18 @@ boro-11
...
+
+If an NVMe SSD is faulty, the status LED on the VMD device will be set to an ON state,
+epresented by a solidly ON amber light.
+This LED activity visually indicates a fault and that the device needs to be replaced and is no
+longer in use by DAOS.
+The LED of the VMD device will remain in this state until replaced by a new device.
 
 !!! note
-    Full NVMe hot plug capability will be available and supported in DAOS 2.2 release.
+    Full NVMe hot plug capability will be available and supported in DAOS 2.6 release.
```
Comment (knard-intel) [inline docs/admin/administration.md:530]: 2.6 ? Not 2.4

Comment (tanabarr) [reply docs/admin/administration.md:530]: 2.6 is correct I believe

Comment (knard-intel) [reply docs/admin/administration.md:530]: OK

**Review by knard-intel — DISMISSED** [review]
LGTM

**Review by rpadma2 — APPROVED** [review]

**Review by knard-intel — APPROVED** [review]
LGTM

**Review by Michael-Hennecke — APPROVED** [review]

**Pull request merged**